packages/styles/forms.css

```diff
@@ -175,6 +175,7 @@ textarea.Field--has-error:hover,
   box-sizing: border-box;
   cursor: pointer;
   border-radius: 3px;
+  color: #333;
```

Comment:
`#333` _is_ a [variable](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/variables.css#L21)...should we use that here?

Reply:
I'd rather have it as a variable, but `--header-text-color-dark` felt too specific. This is an area where I wish we had a palette of variable colors.